### ISSUE TITLE
Question about error in reading in data after the update to v5

### ISSUE THREAD

**Issue by MarioCHMGF** [satijalab/seurat#7272]
Hi, as I was previously reporting this issue and I would really like to know why the following is happening. If I load Seurat < v4.9.9 I can successfully read in data from multiplex 10xG assay and I can successfully create Seurat object. When updating to v4.9.9 i get the following error: 
```
data <- Read10X(data.dir = "//count/sample_filtered_feature_bc_matrix/") 
10X data contains more than one type and is being returned as a list containing matrices of each type.
dataObj <- CreateSeuratObject(counts = data[["Gene Expression"]], project = "test", min.cells = 3, min.features = 200) 
Error in .local(x, na.rm, dims, ...) : object 'lgCMatrix_colSums' not found. 
```
However, I could just use the lower versions, but I can't since I would like to use Azimuth annotations and the later doesn't load without Surat v5.
Does anyone have any recommendation on this issue?
Thank you very much in advance!
With kind regards,
Mario

**Comment by Gesmira** [comment]
Hi Mario, 
Had you used the same object previously in v4 or just other data with multiple assays? Trying to figure out if this is specifically a v5 issue. Furthermore, what is included in the output of `data` or do you have the data available for me to try to reproduce the issue?

In the meantime, you can load the non-v5 version of Azimuth if you downgrade all of your packages. 

```
install.packages(c("SeuratObject", "Seurat"))
library(SeuratObject)
library(Seurat
devtools::install_github('satijalab/azimuth', ref = 'master')
library(Azimuth)
```
should let you use Azimuth normally.

**Comment by MarioCHMGF** [comment]
Dear Gesmira,
Thank you very much for your answer. Yes, I was using the same object and everything was working fine. Of course, I can provide the data, so you can try to reproduce the error. With the recommended Azimuth install I get the error that Seurat Object v4.9.9 is required, but when upgrading, I am again facing the same issue with creating the object. So maybe the issue is more related to the SeuratObject than to Seurat? What would be the most convenient way to share the data with you?
Thank you very much in advance!
With kind regards,
Mario

**Comment by Gesmira** [comment]
Thanks for the info! You can email it to me at: gmolla@nygenome.org

In regards to installing the older version of Azimuth, it's possible that you have other dependencies loaded which require the v5 version of SeuratObject. (SeuratDisk, SeuratData, etc.)

**Comment by MarioCHMGF** [comment]
Dear Gesmira,
I have just sent you an email with the requested data. For Azimuth installation I made sure I have no other dependencies loaded. However, for now, I found a workaround to use the web based Azimuth in the beginning and later I just add the metadata. I have tested it with previous annotations and objects and it works the same.
```
[1] SeuratObject_4.1.3 Seurat_4.3.0
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) :
  namespace ‘SeuratObject’ 4.1.3 is already loaded, but >= 4.9.9.9060 is required
```
Thank you very much in advance for all of your assistance!
With kind regards,
Mario

**Comment by Gesmira** [comment]
Hi Mario, 
Thanks for sending me your data. I am actually able to load it in and work with it fine using the code you've included: 

```
> dataObj <- CreateSeuratObject(counts = data[["Gene Expression"]], project = "test", min.cells = 3, min.features = 200) 
> dataObj
An object of class Seurat 
24062 features across 5275 samples within 1 assay 
Active assay: RNA (24062 features, 0 variable features)
 2 layers present: counts, data
> dataObj <- NormalizeData(dataObj)
Performing log-normalization
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
```

It's possible you have an older version of the v5 package loaded in. Can you clear and restart your R session and run the following: 
```
remotes::install_github("satijalab/seurat", "seurat5", quiet = TRUE)

**Comment by MarioCHMGF** [comment]
Dear Gesmira,

Thank you very much for all of your help. I purged the packages and installed as recommended, now everything works perfectly fine!

I tried several ways to identify the problem and the solution was to install SeuratObject from mojaveazure as suggested by you.

It's funny with SeuratObject from the initial v5 installation it didn't work (or the updated SeuratObject during the installation).

Thank you so much again for all the help!

With kind regards,

Mario

**Comment by MarioCHMGF** [comment]
Dear Gesmira,
Thank you very much for all of your help. I purged the packages and installed as recommended, now everything works perfectly fine!
I tried several ways to identify the problem and the solution was to install SeuratObject from mojaveazure as suggested by you.
It's funny with SeuratObject from the initial v5 installation it didn't work (or the updated SeuratObject during the installation).
Thank you so much again for all the help!
With kind regards,
Mario
